javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,39 @@
+var restart = document.querySelector('button');
+var typer = document.getElementById('phrase');
+var arr = typer.textContent.split('');
```

Comment:
You can store the phrase in a variable here in javascript, and just generate the spans and append when the page loads, so you don't need to get your data from an HTML element before you can do the work you want to do.